### PR TITLE
mon/FSCommands: restore idempotent behavior of fs rm

### PR DESCRIPTION
**Pull request by batrick**
Fixes: https://tracker.ceph.com/issues/48808
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by rishabh-d-dave — APPROVED** [review]
IMO it would be nice to specify in commit message, ticket and/or PR, that we are specifically changing the return value to make the behaviour idempotent. Looks good otherwise and testing with vstart cluster locally went well.

File: qa/tasks/ceph_manager.py [hunk 1]
```diff
@@ -1326,6 +1327,17 @@ def tmp(x):
             except CommandFailedError:
                 self.log('Failed to get pg_num from pool %s, ignoring' % pool)
 
+    def ceph(self, cmd, **kwargs):
+        """
+        Simple Ceph admin command wrapper around run_cluster_cmd.
+        """
+
+        kwargs.pop('args', None)
+        args = shlex.split(cmd)
```
[the remark this answers is not in the document]
Comment (batrick) [reply qa/tasks/ceph_manager.py:1336]: Enforcing a string is kinda what I'm going for in this interface. Specifying lists of strings gets very tiresome so I'm aiming for an interface that requires consistency.

**Comment by batrick** [comment]
> IMO it would be nice to specify in commit message, ticket and/or PR, that we are specifically changing the return value to make the behaviour idempotent. 

The behavior was always idempotent. It was broken by 811e48d6ffd841ecd93a57eb00c6aceae476f4f8. I've updated the commit message to note that.

**Comment by batrick** [comment]
jenkins test make check

**Comment by batrick** [comment]
jenkins test api

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2021-01-12_22:03:59-fs-wip-pdonnell-testing-20210112.170550-distro-basic-smithi/

**Pull request merged**